english/vote-the-president/trump/trump.md

```diff
@@ -8,7 +8,7 @@ Know about [Latest hit from Trump Floyd "The Wall"](./trump-floyd-the-wall/thewa
 
 You must have a tiny.. pair of hands.
 
+Ashamed at your poor electoral choice, and fearful of the consequences of Trump's Presidency, you decide to flee the country.  
 
-
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.